config/settings.yml

```diff
@@ -80,6 +80,8 @@
   :scan_via_host: true
   :use_vim_broker: true
   :use_vim_broker_ems: true
+:container_logging:
+  :common_logging_route: logging-kibana-ops
```

Comment:
@enoodle @Fryguy if we don't want to pollute the `settings.yml` (because anyway a region-wide setting is not making sense) then let's make it a constant and plan to move this to a per-provider setting (advanced setting?).